### DISCUSSION TITLE
Writing a Paper with 2.0.0 Release

### DISCUSSION THREAD

**Discussion by CSSFrancis** [hyperspy/hyperspy#3176] [Polls]
On one hand I really like the idea of __not__ writing a paper for every release because people should just cite the zenodo.  On the other hand people are terrible at doing that and it seems like a decent number of people either:

1. Don't cite hyperspy even if they use it (probably a fairly large number)
2. Cite the M&M abstract from 2017.

This is kind of how it goes as projects get larger and larger (I have yet to see someone in the microscopy community[ cite numpy](https://numpy.org/citing-numpy/).  I don't think hyperspy is quite to that level where it exists post citation.  For getting grants and the such it would be nice to make things more standard.

I like [astropy's](https://www.astropy.org/acknowledging.html) publication as it is more community focused and talks about usage and future plans etc.   I really like the idea of publishing in the [Journal of Open Source Software ](https://joss.theoj.org/) as it includes a code review following these [points ](https://joss.readthedocs.io/en/latest/review_criteria.html#review-items)

@ericpre and @francisco-dlp This might be worth making as a discussion item once #3153 is sorted out.

**Comment by ericpre** [comment]
Thank you @CSSFrancis, these are good points. I don't have time at the moment to look at this and I would need to read more about astropy's approach with respect to citation before commenting!

**Reply by CSSFrancis** [reply]
@ericpre whenever you get around to it!  I liked astropy's publication because they included many of the things @francisco-dlp usually includes in his `hyperspy` talk.  Usage around the world, why to contribute to one opensource project, general philosophy etc.

The sake of getting something out there and maybe publishing with each new major release, the Journal of Open Source Software is another really good option.  I really like their mission and like buying into the idea that published software should be reviewed!  Astropy published a V5.0 paper a v2.0 and a v0.2 paper so they aren't the greatest model for consistency.  Part of that is that writing a paper takes a long time.  Hopefully something like JOSS fixes that.

**Comment by hakonanes** [comment]
I tend to cite the M&M paper from 2017 ([doi](https://doi.org/10.1017/S1431927617001751)) when I reference HyperSpy. When a project does not have a (conference) paper, I cite the Zenodo DOI. This is not the best approach, since not all contributors to HyperSpy are authors of the 2017 paper, for example...

Had a quick look at the AstroPy papers. I like the approach they have taken in focusing on the community/contributions, package development and the package ecosystem they have created.

One route could be to go for a JOSS paper. But, if there is enough interest in writing after it has started, one could instead send it to another journal accepting longer papers. The paper should be open-access.

**Reply by hakonanes** [reply]
My vote in the poll goes for 3., then 2. if there is enough interest.

**Comment by CSSFrancis** [comment]
As we start to get close to a 2.0.0 release we should maybe come back to this.  It might also be a good time to use the new governance model once we have that ratified and merged :)

**Comment by CSSFrancis** [comment]
Just wondering if anyone has thoughts on this now that a 2.0.0 release is imminent. I realize that the time required to make a full publication is a bit of an issue.  If we want to go the route of a complete 2.0.0 paper I would propose that we:

1. Create a detailed outline 
2. Divide the writing of sections to different contributors 
3. Go through a round of heavy editing to refine style etc. and make things more coherent 


We also might want to think about what the goals of a publication are.  If we just want a citable object that isn't a zonodo doi then something like the JOSS should be the route to go. 

The reason for a full length publication would be to explain a larger vision to the scientific community and to (hopefully) reach a wider audience who might want to buy into the concept of hyperspy as a multidimensional analysis tool. 

@ericpre @francisco-dlp any additional thoughts?

**Comment by CSSFrancis** [comment]
@ericpre @francisco-dlp @jlaehne 

I'm at M&M this week and talked to Colin Ophus about his new journal.

https://www.elementalmicroscopy.org


It might be a good place to publish if we wanted. You can also update author lists and have interactive figures.

There is a session later today I am going to attend and will try to learn more about the process.
